view/adminhtml/web/js/validation.js

```diff
@@ -0,0 +1,43 @@
+require([
+        'jquery',
+        'Magento_Ui/js/modal/alert'
+    ],
+    function($, alert){
+        var apiMode = $("#payment_us_breadcheckout_api_mode").val(),
+            prodKey = "#payment_us_breadcheckout_api_public_key",
+            prodSecret = "#payment_us_breadcheckout_api_secret_key",
+            sandKey = "#payment_us_breadcheckout_api_sandbox_public_key",
+            sandSecret = "#payment_us_breadcheckout_api_sandbox_secret_key",
+            selector = [prodKey,prodSecret,sandKey,sandSecret].join(", "),
+            validationUrl = window.location.origin + "/admin/breadadmin/bread/validateCredentials";
+
+        $(selector).on("input",function () {
```

Comment:
When I tried to test this locally it didn't look like this function was ever getting hit when I tried inputing things in the api key textbox

Reply:
It doesn't trigger if secret key is not changed too, reason for that is you can't get secret key value since it's obfuscated

Reply:
I just tried again and changed both the sandbox pub and secret keys, and it still wasn't triggered, any ideas why?